plugin/parser/utils.go

```diff
@@ -67,3 +69,60 @@ func SetLineForMySQLCreateTableStmt(node *tidbast.CreateTableStmt) error {
 	firstLine := node.OriginTextPosition() - strings.Count(node.Text(), "\n")
 	return newTokenizer(node.Text()).setLineForMySQLCreateTableStmt(node, firstLine)
 }
+
+// ExtractTiDBUnsupportStmts returns a list of unsupported statements in TiDB extracted from the `stmts`,
+// and returns the remaining statements supported by TiDB from `stmts`.
+func ExtractTiDBUnsupportStmts(stmts string) ([]string, string, error) {
+	var unsupportStmts []string
+	var supportedStmts string
```

Comment:
use buffer for supportedStmts.

Reply:
How about record them as []string and use strings.Join() to get final string

Reply:
That works too.